### ISSUE TITLE
No Picture In Picture button on playing videos

### ISSUE THREAD

**Issue by SlowNicoFish** [brave/brave-browser#14467]
## Description 

I would love a button on top of a playing video to easily go into PIP mode.
Vivaldi and Firefox has this feature and I would like to have it in Brave as well.
This feature would make it so much more convenient to get into Picture In Picture mode since i use it almost all of the time.

## Steps to Reproduce

   1. Start a video

## Actual result:

I see no button for PIP

## Expected result:
A little button to pop a video out in PIP-mode

## Reproduces how often: 

Always

## Version/Channel Information:

- Can you reproduce this issue with the current release?  Yes
- Can you reproduce this issue with the beta channel?  Yes
- Can you reproduce this issue with the nightly channel?  Yes

## Other Additional Information:

- Does the issue resolve itself when disabling Brave Shields?  No
- Does the issue resolve itself when disabling Brave Rewards?  No
- Is the issue reproducible on the latest version of Chrome?  Yes

**Comment by rebron** [comment]
@SlowNicoFish At the top right corner, there's a global media control UI that will let you get into PIP mode. Does that work for you?

See image below for the control. 

<img width="462" alt="Screen Shot 2021-03-05 at 4 14 57 PM" src="https://user-images.githubusercontent.com/5951041/110187684-0ecd4280-7dce-11eb-93e3-956570f01fbf.png">

**Comment by SlowNicoFish** [comment]
It's not really optimal because it throws off the flow i have from other browser with that feature. Vivaldi and Firefox has that feature I think that it really adds to the experience.

**Comment by rebron** [comment]
An overlay button on the video to get into PIP mode. There is redundancy here since there is a way to get into PIP mode, just not as discoverable.

<img width="162" alt="Screen Shot 2021-05-07 at 3 40 13 PM" src="https://user-images.githubusercontent.com/5951041/117515297-b0961a80-af4a-11eb-87b5-0531c186f395.png">

**Comment by Haravikk** [comment]
> @SlowNicoFish At the top right corner, there's a global media control UI that will let you get into PIP mode. Does that work for you?

Thanks for mentioning this! I wasn't aware of it at all; I think discoverability is a bit poor as it's not super obvious, despite being super useful.

In my case I'm used to right-clicking (or double right-clicking for sites like YouTube with custom right-click menus) on a video in other browsers to open it in Picture-in-Picture or Fullscreen.

At the very least it might be wroth Brave adding PiP/Fullscreen options to the right-click menu for videos? It currently just seems to show the same menu as for anywhere else on the page.

**Comment by SlowNicoFish** [comment]
Now even Microsoft edge browser has implemented it.
@Haravikk Yes, that would be a good solution, not great but good enough. The best would of course be a floating button

**Comment by Mennaruuk** [comment]
Yup. Edge, Firefox, and Vivaldi have this feature. Brave will be a more perfect browser with this button!

**Comment by nonokh** [comment]
Please add media scrubber like in Firefox. It makes absolute no sense to go back to tab just to seek 10 secs -/+

**Comment by SlowNicoFish** [comment]
Yes, the pip functionality needs to be much improved. Take a look at Firefox and Vivaldi for inspiration
